app/views/my_facilities/missed_visits.html.erb

```diff
@@ -167,6 +167,30 @@
             </tbody>
           </table>
         <% end %>
+        <div class="d-flex mt-4">
+          <em class="h-12px w-12px mr-8px bg-red br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            &gt;30%
+          </p>
+        </div>
+        <div class="d-flex">
+          <em class="h-12px w-12px mr-8px bg-orange br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            21-30%
+          </p>
+        </div>
+        <div class="d-flex">
+          <em class="h-12px w-12px mr-8px bg-yellow br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            11-20%
+          </p>
+        </div>
+        <div class="d-flex">
+          <em class="h-12px w-12px mr-8px bg-green-medium br-6px"></em>
+          <p class="fs-12px lh-1 c-grey-dark">
+            0-10%
+          </p>
+        </div>
```

Comment:
Look at `BP not controlled` comment: https://github.com/simpledotorg/simple-server/pull/3624/files#r831377196

Reply:
Done